diff --git a/node_modules/bun-types/.DS_Store b/.DS_Store
new file mode 100644
index 0000000000000000000000000000000000000000..3257b1909382d16a199b9017e1794a82e7c193a5
Binary files /dev/null and b/.DS_Store differ
diff --git a/node_modules/bun-types/docs/.DS_Store b/docs/.DS_Store
new file mode 100644
index 0000000000000000000000000000000000000000..9f3015676822bb4b85fbc72c0cf5833f42f54c0c
Binary files /dev/null and b/docs/.DS_Store differ
diff --git a/package.json b/package.json
index 199e2554e1ee87dde46f331f6207d10148de8be7..91c3e8492d126f9c71ad84c444ac3f7a025018c2 100644
--- a/package.json
+++ b/package.json
@@ -20,7 +20,8 @@
   ],
   "homepage": "https://bun.com",
   "dependencies": {
-    "@types/node": "*"
+    "@types/node": "*",
+    "undici-types": "*"
   },
   "scripts": {
     "prebuild": "echo $(pwd)",
